ci/setup_env.sh

```diff
@@ -36,9 +36,13 @@ else
   exit 1
 fi
 
-wget -q "https://repo.continuum.io/miniconda/Miniconda3-latest-$CONDA_OS.sh" -O miniconda.sh
+if [ "${TRAVIS_CPU_ARCH}" == "arm64" ]; then
+  wget -q "https://github.com/conda-forge/miniforge/releases/download/4.8.2-1/Miniforge3-4.8.2-1-Linux-aarch64.sh" -O miniconda.sh;
+else
+  wget -q "https://repo.continuum.io/miniconda/Miniconda3-latest-$CONDA_OS.sh" -O miniconda.sh;
+fi
 chmod +x miniconda.sh
-./miniconda.sh -b
+./miniconda.sh -b -p $MINICONDA_DIR
```

Comment:
why this change? u should already be in the correct dir

Reply:
In the case of the arm64 platform, I am using miniforge3 to install conda. When we run miniconda.sh(Miniforge3-4.8.2-1-Linux-aarch64.sh) for arm64 platform, it will get installed in $HOME/miniforge3 folder whereas for x64 platform it will get installed in $HOME/miniconda3. So, I am providing the installation path as $HOME/miniconda3.

Reply:
May be you can you add a short comment before this line, so this information is explicit in the code.